### ISSUE TITLE
500 Server Error stopped at  ( 64 / 247 ) 

### ISSUE THREAD

**Issue by rgleason** [i-net-software/dokuwiki-plugin-siteexport#69]
SITEEXPORT PLUGIN

Uses IndexMenu Plugin for Left Navigation. Turned off all Javascript for the download. See settings below

This line stopped making the 'siteexport.zip'  
The progress indicator underneath stopped and a 500 Error appeared in Firebug.

Adding "opencpn:opencpn_user_manual:charts:chart_sources" ( 64 / 247 ) 

Using Firefox  51.0.1 (32-bit)  with Firebug   2.0.91  turned on in the menu (yellow).
I also have "Debug checked under ADMIN > CONFIG   Then use firefox "File/find" and search on Debug
  and check it.  (Should be only temporary check, turn off afterwards) 

See attached please.
[500 Internal Error-w.docx](https://github.com/i-net-software/dokuwiki-plugin-siteexport/files/814019/500.Internal.Error-w.docx)

![screenshot 57](https://cloud.githubusercontent.com/assets/2363959/23510781/e4f21396-ff28-11e6-9d36-f47b044ab04a.png)
![screenshot 59](https://cloud.githubusercontent.com/assets/2363959/23510807/f8b89472-ff28-11e6-9be7-b42d0c4ca271.png)

Gerry do you have any suggestions?  Thanks.

**Comment by rgleason** [comment]
The above Firefox Firefly debugging is useful, but the Debugging feature within SiteExport is more informative.  We have made good notes on how to get SiteExport Debug working, see the 2 links below which might provide useful documentation for others.

We finally got debugging working, but are still working on getting the SiteExport to complete.. See #70 #71 #72   #73.   We will be closing this.

The lead developer has an issue he posted. #74 

=====
Set Namespace: opencpn:opencpn_user_manual:toc <- Point directly to the TOC Page.
Set Parent Namespace: opencpn:opencpn_user_manual <- Point directly to the User Manual namespace.

Here is a documentation page for the OpenCPN.org "Books" Indexmenu Plugin showing how to build a
TOC page for a namespace that can be used in SiteExport.

https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn:opencpn_user_manual:edit_user_manual:test_navigation_toc#siteexport_pluginopencpn_user_manual_toc

Here is a documentation page for the OpenCPN.org "Books" SiteExport Plugin showing how to use the TOC page in SiteExport and what configuration settings we use.
https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn:opencpn_user_manual:edit_user_manual:site_export_plugin

There is also information about setting up the debug.txt file in case the plugin is not working.

We are still trying to get it to complete, however.
